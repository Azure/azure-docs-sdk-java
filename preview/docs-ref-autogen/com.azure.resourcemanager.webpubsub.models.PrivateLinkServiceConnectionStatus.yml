### YamlMime:JavaType
uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus"
fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
  name: "PrivateLinkServiceConnectionStatus()"
  nameWithType: "PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
  syntax: "public PrivateLinkServiceConnectionStatus()"
fields:
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.DISCONNECTED"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.DISCONNECTED"
  name: "DISCONNECTED"
  nameWithType: "PrivateLinkServiceConnectionStatus.DISCONNECTED"
  summary: "Static value Disconnected for PrivateLinkServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateLinkServiceConnectionStatus DISCONNECTED"
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.PENDING"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.PENDING"
  name: "PENDING"
  nameWithType: "PrivateLinkServiceConnectionStatus.PENDING"
  summary: "Static value Pending for PrivateLinkServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateLinkServiceConnectionStatus PENDING"
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.REJECTED"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.REJECTED"
  name: "REJECTED"
  nameWithType: "PrivateLinkServiceConnectionStatus.REJECTED"
  summary: "Static value Rejected for PrivateLinkServiceConnectionStatus."
  modifiers:
  - "static"
  - "final"
  field:
    type: "<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static final PrivateLinkServiceConnectionStatus REJECTED"
methods:
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.fromString(String name)"
  name: "fromString(String name)"
  nameWithType: "PrivateLinkServiceConnectionStatus.fromString(String name)"
  summary: "Creates or finds a PrivateLinkServiceConnectionStatus from its string representation."
  modifiers:
  - "static"
  parameters:
  - description: "a name to look for."
    name: "name"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public static PrivateLinkServiceConnectionStatus fromString(String name)"
  returns:
    description: "the corresponding PrivateLinkServiceConnectionStatus."
    type: "<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />"
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.values()"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.values()"
  name: "values()"
  nameWithType: "PrivateLinkServiceConnectionStatus.values()"
  modifiers:
  - "static"
  syntax: "public static Collection<PrivateLinkServiceConnectionStatus> values()"
  returns:
    description: "known PrivateLinkServiceConnectionStatus values."
    type: "<xref href=\"java.util.Collection?alt=java.util.Collection&text=Collection\" data-throw-if-not-resolved=\"False\" />&lt;<xref href=\"com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus?alt=com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus&text=PrivateLinkServiceConnectionStatus\" data-throw-if-not-resolved=\"False\" />&gt;"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.webpubsub.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-webpubsub:1.0.0-beta.2
