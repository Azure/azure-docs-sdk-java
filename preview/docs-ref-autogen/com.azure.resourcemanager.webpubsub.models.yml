### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.webpubsub.models"
fullName: "com.azure.resourcemanager.webpubsub.models"
name: "com.azure.resourcemanager.webpubsub.models"
summary: "Package containing the data models for WebPubSubManagementClient."
classes:
- "com.azure.resourcemanager.webpubsub.models.AclAction"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificateList"
- "com.azure.resourcemanager.webpubsub.models.CustomDomainList"
- "com.azure.resourcemanager.webpubsub.models.Dimension"
- "com.azure.resourcemanager.webpubsub.models.EventHandler"
- "com.azure.resourcemanager.webpubsub.models.EventHubEndpoint"
- "com.azure.resourcemanager.webpubsub.models.EventListener"
- "com.azure.resourcemanager.webpubsub.models.EventListenerEndpoint"
- "com.azure.resourcemanager.webpubsub.models.EventListenerEndpointDiscriminator"
- "com.azure.resourcemanager.webpubsub.models.EventListenerFilter"
- "com.azure.resourcemanager.webpubsub.models.EventListenerFilterDiscriminator"
- "com.azure.resourcemanager.webpubsub.models.EventNameFilter"
- "com.azure.resourcemanager.webpubsub.models.KeyType"
- "com.azure.resourcemanager.webpubsub.models.LiveTraceCategory"
- "com.azure.resourcemanager.webpubsub.models.LiveTraceConfiguration"
- "com.azure.resourcemanager.webpubsub.models.LogSpecification"
- "com.azure.resourcemanager.webpubsub.models.ManagedIdentity"
- "com.azure.resourcemanager.webpubsub.models.ManagedIdentitySettings"
- "com.azure.resourcemanager.webpubsub.models.ManagedIdentityType"
- "com.azure.resourcemanager.webpubsub.models.MetricSpecification"
- "com.azure.resourcemanager.webpubsub.models.NameAvailabilityParameters"
- "com.azure.resourcemanager.webpubsub.models.NetworkAcl"
- "com.azure.resourcemanager.webpubsub.models.OperationDisplay"
- "com.azure.resourcemanager.webpubsub.models.OperationList"
- "com.azure.resourcemanager.webpubsub.models.OperationProperties"
- "com.azure.resourcemanager.webpubsub.models.PrivateEndpoint"
- "com.azure.resourcemanager.webpubsub.models.PrivateEndpointAcl"
- "com.azure.resourcemanager.webpubsub.models.PrivateEndpointConnectionList"
- "com.azure.resourcemanager.webpubsub.models.PrivateLinkResourceList"
- "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus"
- "com.azure.resourcemanager.webpubsub.models.ProvisioningState"
- "com.azure.resourcemanager.webpubsub.models.RegenerateKeyParameters"
- "com.azure.resourcemanager.webpubsub.models.ResourceLogCategory"
- "com.azure.resourcemanager.webpubsub.models.ResourceLogConfiguration"
- "com.azure.resourcemanager.webpubsub.models.ResourceReference"
- "com.azure.resourcemanager.webpubsub.models.ResourceSku"
- "com.azure.resourcemanager.webpubsub.models.ScaleType"
- "com.azure.resourcemanager.webpubsub.models.ServiceSpecification"
- "com.azure.resourcemanager.webpubsub.models.ShareablePrivateLinkResourceProperties"
- "com.azure.resourcemanager.webpubsub.models.ShareablePrivateLinkResourceType"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResourceList"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResourceStatus"
- "com.azure.resourcemanager.webpubsub.models.SignalRServiceUsageList"
- "com.azure.resourcemanager.webpubsub.models.SignalRServiceUsageName"
- "com.azure.resourcemanager.webpubsub.models.Sku"
- "com.azure.resourcemanager.webpubsub.models.SkuCapacity"
- "com.azure.resourcemanager.webpubsub.models.UpstreamAuthSettings"
- "com.azure.resourcemanager.webpubsub.models.UpstreamAuthType"
- "com.azure.resourcemanager.webpubsub.models.UserAssignedIdentityProperty"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHubList"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHubProperties"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubNetworkACLs"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubRequestType"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResourceList"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubSkuTier"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubTlsSettings"
interfaces:
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.Definition"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.Blank"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.WithKeyVaultBaseUri"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.WithKeyVaultSecretName"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.WithKeyVaultSecretVersion"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.Update"
- "com.azure.resourcemanager.webpubsub.models.CustomCertificate.UpdateStages"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.Definition"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages.Blank"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages.WithCustomCertificate"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages.WithDomainName"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.Update"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.UpdateStages"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.UpdateStages.WithCustomCertificate"
- "com.azure.resourcemanager.webpubsub.models.CustomDomain.UpdateStages.WithDomainName"
- "com.azure.resourcemanager.webpubsub.models.NameAvailability"
- "com.azure.resourcemanager.webpubsub.models.Operation"
- "com.azure.resourcemanager.webpubsub.models.Operations"
- "com.azure.resourcemanager.webpubsub.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.webpubsub.models.PrivateLinkResource"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.Definition"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.Blank"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.WithGroupId"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.WithPrivateLinkResourceId"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.DefinitionStages.WithRequestMessage"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.Update"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.UpdateStages"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.UpdateStages.WithGroupId"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.UpdateStages.WithPrivateLinkResourceId"
- "com.azure.resourcemanager.webpubsub.models.SharedPrivateLinkResource.UpdateStages.WithRequestMessage"
- "com.azure.resourcemanager.webpubsub.models.SignalRServiceUsage"
- "com.azure.resourcemanager.webpubsub.models.SkuList"
- "com.azure.resourcemanager.webpubsub.models.Usages"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubCustomCertificates"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubCustomDomains"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.Definition"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.DefinitionStages"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.DefinitionStages.Blank"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.Update"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.UpdateStages"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHub.UpdateStages.WithProperties"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubHubs"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubKeys"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubPrivateEndpointConnections"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubPrivateLinkResources"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.Definition"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.Blank"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithDisableAadAuth"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithDisableLocalAuth"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithLiveTraceConfiguration"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithNetworkACLs"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithResourceLogConfiguration"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithSku"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithTags"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.DefinitionStages.WithTls"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.Update"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithDisableAadAuth"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithDisableLocalAuth"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithLiveTraceConfiguration"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithNetworkACLs"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithResourceLogConfiguration"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithSku"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithTags"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubResource.UpdateStages.WithTls"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubSharedPrivateLinkResources"
- "com.azure.resourcemanager.webpubsub.models.WebPubSubs"
desc: "Package containing the data models for WebPubSubManagementClient. REST API for Azure WebPubSub Service."
metadata: {}
package: "com.azure.resourcemanager.webpubsub.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-webpubsub:1.0.0-beta.3
