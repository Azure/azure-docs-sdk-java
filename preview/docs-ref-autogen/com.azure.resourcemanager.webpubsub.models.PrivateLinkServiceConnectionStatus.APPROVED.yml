### YamlMime:JavaMember
uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.webpubsub.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateLinkServiceConnectionStatus."
type: "field"
metadata: {}
package: "com.azure.resourcemanager.webpubsub.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-webpubsub:1.0.0-beta.2
